src/core/public/application/integration_tests/application_service.test.tsx

```diff
@@ -106,9 +106,13 @@ describe('ApplicationService', () => {
           },
         });
 
-        const { navigateToApp, currentAppId$ } = await service.start(startDeps);
+        const { navigateToApp, currentAppId$, getComponent } = await service.start(startDeps);
+        update = createRenderer(getComponent());
 
-        await act(() => navigateToApp('app1'));
+        await act(async () => {
+          await navigateToApp('app1');
```

Comment:
nit: update test description because we no longer emit before mounting but after

Reply:
in the unit test we removed we tested several calls to `navigateToApp` and then checked all the events on currentAppId$

Because this test uses `take(1)` it doesn't for instance test that there's just a single event per navigate. So now that we've removed the unit tests I think it's worth expanding this one slightly.

Reply:
> nit: update test description because we no longer emit before mounting but after

We technically still emit **before**  calling the application's `mount` function. The naming of the tests and the distinction was just very misleading.